src/services/request-limit-record-service.js

```diff
@@ -0,0 +1,50 @@
+import Logger from '../common/logger.js';
+import { createShortHash } from '../utils/string-utils.js';
+import RequestLimitRecordStore from '../stores/request-limit-record-store.js';
+
+const logger = new Logger(import.meta.url);
+
+const FAILED_LOGIN_ATTEMPTS_THRESHOLD = 5;
+const FAILED_LOGIN_ATTEMPTS_KEY = 'failed-login-attempts';
+const FAILED_LOGIN_ATTEMPTS_TIME_WINDOW_IN_MS = 15 * 60 * 1000;
```

Comment:
I know these are used juts here, but maybe it would be handy to collect all such setup in the `constants` module. We already have similar constants there like `PENDING_USER_REGISTRATION_EXPIRATION_IN_HOURS`

Reply:
Good point. Moved them to the `constants` module.